### PR TITLE
Attendee mailer

### PR DESCRIPTION
**Pull request by mmdotz**
changes to HTML document route to rsvp only

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -16,7 +16,6 @@
   get '/users' => 'users#index'
   get '/pdf' => 'events#index', as: 'pdf'
   get '/events_csv' => 'events#index', as: 'events_csv'
-  get '/rsvp' => 'application#rsvp'
```
Comment (cmccammon) [inline config/routes.rb:19]: did you create a new route for rsvp?

Comment (mmdotz) [inline config/routes.rb:19]: no it was reset in anisha's code to simply route to update

Comment (mmdotz) [inline config/routes.rb:19]: i think her pull request was completed and merged

**Comment by mmdotz** [comment]
chris did that

**Pull request merged**